src/main/java/oof/logic/command/productivity/TrackerCommand.java

```diff
@@ -91,6 +89,9 @@ public void execute(SemesterList semesterList, TaskList taskList, Ui ui, Storage
                 throw new InvalidArgumentException("Invalid Commmand!");
             }
             int deleteIndex = Integer.parseInt(input[DELETE_INDEX]) - CORRECT_INDEX;
+            if (deleteIndex >= taskList.getSize()) {
+                throw new InvalidArgumentException("Invalid Commmand!");
+            }
             tracker = trackerList.get(deleteIndex);
```

Comment:
Can you make the error message more meaningful? E.g. Please enter a valid index! 
Or something along those lines cos it may be marked as a low level bug thanks.